config/software/autoconf.rb

```diff
@@ -24,7 +24,8 @@
 
 dependency "m4"
 
-version("2.69") { source sha256: "954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969" }
+version("2.71") { source sha256: "431075ad0bf529ef13cb41e9042c542381103e80015686222b8a9d4abef42a1c" }
```

Comment:
Can we add the `# version_list:` comment here?

Reply:
Updated the list of all the available versions for autoconf